eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -133,7 +133,7 @@ stages:
         PreRunSteps:
         - template: /eng/common/pipelines/templates/steps/cosmos-emulator.yml
           parameters:
-            StartParameters: '-EnableAadAuthentication -PartitionCount 50 -Consistency Strong -Timeout 600 -EnablePreview'
+            StartParameters: '/noexplorer /noui /enablepreview /disableratelimiting /enableaadauthentication /partitioncount=50 /consistency=Strong'
```

Comment:
Unrelated to current PR (just about emulator)

Convention of "-" and "/": are these two separate conventions supported? 
If not won;t it break existing flows? 

/cc: @milismsft

Reply:
"-" is used for the Powershell script (that we used internally but does not yet allow specifying the location of the actual binary) - "/" is used by the executable itself.

Ideally we will add a parameter to the Powersell scripts allowing to specify the binary location - then switch back to use the original powershell script to start etc. the emulator. @milismsft - agreed?